.github/ISSUE_TEMPLATE/TASK.yaml

```
@@ -1,11 +1,75 @@
----
-name: 🏃🏻‍♂️ Task
-description: A general task that does not fit into other categories.
-title: "[task]: "
-type: Task
+name: "Task"
+description: "Work item that goes through refinement before development"
+title: "[Task] <short title>"
+labels: ["task"]
+
 body:
   - type: textarea
-    id: description
+    id: purpose
     attributes:
-      label: Description
-      description: Give a brief description of the task.
+      label: Purpose
+      description: Briefly explain why this task is important and what it delivers.
+      placeholder: |
+        Briefly describe what the task should deliver and why it matters.
+        e.g. Add caching to CI pipelines to reduce build time.
+    validations:
+      required: true
+
+  # --- Do we want to estimate the size of the task? If yes, this is my suggestion,
+  # since we used T-shirt sizes in the old DevOps team and it worked well.
+  # This combines that approach with a bit of planning poker. ---
+  - type: dropdown
+    id: size
+    attributes:
+      label: Size (T-shirt)
+      options: [XS, S, M, L, XL]
+    validations:
+      required: true
```

```suggestion
  - type: dropdown
    id: scope
    attributes:
      label: Scope
      options: [S, M, L]
```

1. I am on board. I like it, and I think it would be a really effective way to concretize peoples thoughts of a task, if we make a quick "around the table" on t-shirt size inspired estimation.
1. I think less is more here. I would rather start small and add more sizes if necessary, than having many options up front.
2. I do not think we should make this required, as it may be hard to define the size of a task at task creation. Ensuring it is not a requirement, allows this to be something that is considered during refinement.